.gitmodules

```diff
@@ -10,3 +7,7 @@
 [submodule "modules/afamqp/rabbitmq-c"]
 	path = modules/afamqp/rabbitmq-c
 	url = https://github.com/alanxz/rabbitmq-c.git
+[submodule "modules/afmongodb/mongo-c-driver"]
```

Comment:
The domain socket fix has been merged under tag 1.2.1 along with the other one:
https://github.com/mongodb/mongo-c-driver/commit/6614693e481b3614a8ae8af2dc10bac2c768a82a